Applications/web_portal/web_app/library/source/pages/account_page/risk_page/risk_parameters_view.tsx

```diff
@@ -8,13 +12,317 @@ interface Properties {
 
   /** Used to lookup currency names and symbols. */
   currencyDatabase: Nexus.CurrencyDatabase;
+
+  /** Called on an update of the currency. */
+  onCurrencyChange?: (currency: Nexus.Currency) => void;
+
+  /** Called on an update of the buying power. */
+  onBuyingPowerChange?: (value: Nexus.Money) => void;
+
+  /** Called on an update of the net loss. */
+  onNetLossChange?: (value: Nexus.Money) => void;
+
+  /** Called on an update of the transition time. */
+  onTransitionTimeChange?: (duration: Beam.Duration) => void;
+}
+
+interface State {
+  breakpoint: Breakpoint;
 }
 
-interface State {}
+interface FooterProperties {
+  isButtonDisabled: boolean;
+  isUserAdmin: boolean;
+  message: FOOTER_MESSAGE;
+  comment?: string;
+  onButtonClick: () => void;
+  onCommentChange: (event: React.ChangeEvent<HTMLTextAreaElement>) => void;
+}
+
+enum Breakpoint {
+  SMALL,
+  MEDIUM,
+  LARGE
+}
+
+enum FOOTER_MESSAGE {
+  SAVED,
+  SUBMITTED,
+  SERVER_ISSUE,
+  NONE
+}
+
+interface LabelProperties {
+  text: string;
+}
+
+interface TransistionTimeProperties {
+  onTransitionTimeUpdate?: (duration: number, field: string) => void;
+}
 
 /** Implements a React component to display a set of RiskParameters. */
 export class RiskParametersView extends React.Component<Properties, State> {
+
+  public constructor(props: Properties) {
+    super(props);
+    this.state = {
+      breakpoint: RiskParametersView.getBreakpoint()
+    };
+    this.onScreenResize = this.onScreenResize.bind(this);
+    this.onTransitionSecondChange = this.onTransitionSecondChange.bind(this);
+    this.onTransitionMinuteChange = this.onTransitionMinuteChange.bind(this);
+    this.onTransitionHourChange = this.onTransitionHourChange.bind(this);
+  }
+
+  public componentDidMount() {
+    window.addEventListener('resize', this.onScreenResize);
+  }
+
+  public componentWillUnmount(): void {
+    window.removeEventListener('resize', this.onScreenResize);
+  }
+
   public render(): JSX.Element {
-    return null;
+    const containerClassName = (() => {
+      switch(this.state.breakpoint) {
+        case Breakpoint.SMALL:
+          return css([RiskParametersView.CONTAINER_STYLE.small,
+            RiskParametersView.CONTAINER_STYLE.base]);
+        case Breakpoint.MEDIUM:
+          return css([RiskParametersView.CONTAINER_STYLE.medium,
+            RiskParametersView.CONTAINER_STYLE.base]);
+        case Breakpoint.LARGE:
+          return css([RiskParametersView.CONTAINER_STYLE.large,
+            RiskParametersView.CONTAINER_STYLE.base]);
+        default:
+          return css([RiskParametersView.CONTAINER_STYLE.medium,
+              RiskParametersView.CONTAINER_STYLE.base]);
+      }
+    })();
+    const hours = Math.floor(
```

Comment:
Use this.props.parameters.transitionTime.split() to get the JSON object containing the hours/minutes/seconds.

Reply:
Use the split method.